ui/pages/asset/components/token-buttons.tsx

```diff
@@ -93,7 +93,6 @@ const TokenButtons = ({
 
   const account = useSelector(getSelectedInternalAccount, isEqual);
 
-  // This is only returning solana mainnet, not solana devnet?
```

Comment:
lol, sorry, this was a dev comment during debugging for https://github.com/MetaMask/metamask-extension/pull/32679